javascript/tables/src/readTable.utils.ts

```diff
@@ -43,19 +43,19 @@ function arqueroParser(text: string, options: ParserOptions = {}): ColumnTable {
 	return options.readRows ? table.slice(0, options.readRows) : table
 }
 
-function papaParser(text: string, options: ParserOptions = {}): ColumnTable {
-	const opts = { ...options }
-	if (opts.skipRows && opts.readRows) {
-		opts.readRows += opts.skipRows
-	}
-	const mappedOptions = mapProps(ParserType.PapaParse, opts) as ParseConfig
-	const table = papa.parse(text, mappedOptions)
-	if (opts.skipRows) {
-		const subset = skipRows(table.data, opts.skipRows)
-		table.data = subset
-	}
-	return from(table.data)
-}
+// function papaParser(text: string, options: ParserOptions = {}): ColumnTable {
```

Comment:
what's going on here?

Reply:
there's a handful of commented out pieces - can we either delete these code blocks or add a comment about why they're commented out?

Reply:
instead of using only fromCSV we need the whole mapping of params from arquero, so I had to re-enable readCsvTable,  but only use arqueroParser on the switch. By commenting the use of papaParser, lint complains that this function is unused if I leave it there alone

Reply:
sure, I'll add a comment